services/file_writers/projects_writer.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require_relative "file_writer"
+
+module FastlaneCI
```

Comment:
I'm going to use the JSONDataSource that we currently have instead of this - this part is currently pretty bad 😳

Reply:
Also, if you want to use file templates, you might want to consider using ERBs (like how we have them for our UI). That being said, if you're using JSON then yah, JSONDataSource might be the way to go 👍

Reply:
Good call! For the `keys` template I'll use ERB :D